android/res/raw/about.html

```diff
@@ -1,87 +1,75 @@
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=benhc123">benhc123</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=Humzini">Humzini</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=grzesiekrzaca">grzesiekrzaca</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=ireun">ireun</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=naofum">naofum</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/frostwire/frostwire/commits?author=muckachina">muckachina</a></font>,
+<h3>Lead Designer</h3>
+<font color="#2e9ec7"><a href="http://marcelinkaaa.com/">Marcelina Knitter</a></font> <font color="#2e9ec7"><a href="http://twitter.com/marcelinkaaa">@marcelinkaaa</a></font>
+<br />
+<h3>Collaborators on github</h3>
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=alejandroarturom">alejandroarturom</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=votaguz">votaguz</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=X-NicON">X-NicON</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=benhc123">benhc123</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=Humzini">Humzini</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=grzesiekrzaca">grzesiekrzaca</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=ireun">ireun</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=naofum">naofum</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/frostwire/frostwire/commits?author=muckachina">muckachina</a></font>,
 
 <br />
-<h2>Testers</h2>
-<font color="#5ca8bf"><a href="https://github.com/muckachina">muckachina</a></font>,
-<font color="#5ca8bf"><a href="https://twitter.com/#!/KG8OC">@KG80C</a></font>,
-<font color="#5ca8bf"><a href="https://github.com/untaimed18">untaimed18</a></font>, Eric O'Brian, William Whitman, travis Rhodes, gccengineering, votaguz and every one testing and complaining.
+<h3>Testers</h3>
+<font color="#2e9ec7"><a href="https://github.com/muckachina">muckachina</a></font>,
+<font color="#2e9ec7"><a href="https://twitter.com/#!/KG8OC">@KG80C</a></font>,
+<font color="#2e9ec7"><a href="https://github.com/untaimed18">untaimed18</a></font>, Eric O'Brian, William Whitman, travis Rhodes, gccengineering, votaguz and every one testing and complaining.
 <br /><br />
 Thanks for complaining and letting us know when we suck, please keep it up.
-<h2>Kudos & Thanks</h2>
+<h3>Kudos & Thanks</h3>
```

Comment:
So I can not seem to find a definite answer on that. This is what we always had and it worked. We could also use &#38; - works well as well just tested. 
Or a simple and safe "and"? ;)